install/pre-install-payload/payload.sh

```diff
@@ -78,21 +78,21 @@ function build_payload() {
 		docker push "${registry}:${kernel_arch}-${tag}"
 	done
 
-	purge_previous_manifests "${registry}:${tag}"
-	purge_previous_manifests "${registry}:latest"
+	purge_previous_manifests ${registry}:${tag}
+	purge_previous_manifests ${registry}:latest
 
-	docker manifest create "${extra_docker_manifest_flags}" \
-		"${registry}:${tag}" \
-		"${registry}:x86_64-${tag}" \
-		"${registry}:s390x-${tag}"
+	docker manifest create ${extra_docker_manifest_flags} \
```

Comment:
here it is a good idea to drop the `${extra_docker_manifest_flags}` quotes but keep the other ones